tests/transport/test_transport.py

```diff
@@ -12,16 +12,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import unittest
+import httplib2
+import pytest
+import urllib3
 
-import mock
+from oauth2client import _transport
+import oauth2client._transport.httplib2
+import oauth2client._transport.urllib3
 
-from oauth2client import transport
 
+def test__get_preferred_http_object():
+    assert isinstance(
+        _transport._get_preferred_http_object(), urllib3.PoolManager)
```

Comment:
No? This tests makes sure that our preferred http object is urllib3 even when all transports are available.